hs_modflow_modelinstance/forms.py

```diff
@@ -0,0 +1,568 @@
+from django.forms import ModelForm
+from django import forms
+from crispy_forms import layout
+from crispy_forms.layout import Layout, HTML
+
+from hs_core.forms import BaseFormHelper, Helper
+from hs_core.hydroshare import users
+
+from hs_modelinstance.models import ModelOutput, ExecutedBy
+from hs_modflow_modelinstance.models import StudyArea, GridDimensions, StressPeriod, \
+    GroundWaterFlow, BoundaryCondition, ModelCalibration, ModelInput, GeneralElements
+
+
+gridTypeChoices = (('Choose a type', 'Choose a type'), ('Regular', 'Regular'),

```

Comment:
Are these ```...Choices```` variables used outside this file? If not, then define these variables inside the respective form classes where there are used. Also will prefer not to use camel casing for variable naming.